From 5425c530ad2ce35411bffd3972cd64c4c94dbc97 Mon Sep 17 00:00:00 2001
From: Pawit Pornkitprasan <p.pawit@gmail.com>
Date: Wed, 25 Dec 2013 19:03:43 +0700
Subject: [PATCH 023/206] Camera2: enable antibanding by default

The default for antibanding should be "auto", otherwise there will
be banding effect when taking photos in certain lighting.

Normally, the default on most devices is "auto" (depends on Camera
HAL). But the code from CAF changed it so that the Camera app
explicitly change it to "off".

Change-Id: Id0c2ef6e17bac74d22e12213ab2ed26c1c73783d
---
 res/values/qcomstrings.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/res/values/qcomstrings.xml b/res/values/qcomstrings.xml
index 0ec619163..9dbe6090c 100755
--- a/res/values/qcomstrings.xml
+++ b/res/values/qcomstrings.xml
@@ -486,7 +486,7 @@
      <string name="pref_camera_antibanding_title">Anti Banding</string>
 
      <!-- Default Anti Banding setting. -->
-     <string name="pref_camera_antibanding_default">off</string>
+     <string name="pref_camera_antibanding_default">auto</string>
 
      <!-- Settings screen, Anti Banding entries -->
      <string name="pref_camera_antibanding_entry_0">Off</string>
-- 
2.17.1

